src/main/java/com/amazon/opendistroforelasticsearch/sql/domain/Field.java

```diff
@@ -35,8 +36,9 @@
 	private String alias;
     private NestedType nested;
     private ChildrenType children;
+    private SQLExpr expression;
```

Comment:
I don't see the use of this new field member. Did I miss it anywhere?

Reply:
see answer below. It's part of upcoming PR that I've slipped here. I can move it into different PR or leave it as is, because the change will be part of next PR anyway